Apps/Sandcastle/gallery/Geometry Height Reference.html

```diff
@@ -0,0 +1,98 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="utf-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=1, minimum-scale=1, user-scalable=no">
+    <meta name="description" content="An example for how to use the GeometryHeightProperty to height reference a corridor, ellipse, polygon or rectangle.">
+    <meta name="cesium-sandcastle-labels" content="Geometries">
+    <title>Cesium Demo</title>
+    <script type="text/javascript" src="../Sandcastle-header.js"></script>
+    <script type="text/javascript" src="../../../ThirdParty/requirejs-2.1.20/require.js"></script>
+    <script type="text/javascript">
+        if(typeof require === "function") {
+            require.config({
+                baseUrl : '../../../Source',
+                waitSeconds : 120
+            });
+        }
+    </script>
+</head>
+<body class="sandcastle-loading" data-sandcastle-bucket="bucket-requirejs.html">
+<style>
+    @import url(../templates/bucket.css);
+</style>
+<div id="cesiumContainer" class="fullSize"></div>
+<div id="loadingOverlay"><h1>Loading...</h1></div>
+<div id="toolbar"></div>
+<script id="cesium_sandcastle_script">
+function startup(Cesium) {
+    'use strict';
+//Sandcastle_Begin
+var cesiumTerrainProvider = Cesium.createWorldTerrain();
+var ellipsoidTerrainProvider = new Cesium.EllipsoidTerrainProvider();
+var viewer = new Cesium.Viewer('cesiumContainer', {
+    baseLayerPicker: false,
+    terrainProvider: cesiumTerrainProvider
+});
+viewer.scene.globe.depthTestAgainstTerrain = true;
+
+Sandcastle.addToolbarButton('Enable Terrain', function() {
+    viewer.scene.terrainProvider = cesiumTerrainProvider;
+});
+
+Sandcastle.addToolbarButton('Disable Terrain', function() {
+    viewer.scene.terrainProvider = ellipsoidTerrainProvider;
+});
+
+
+var baseLon = 6.850615989890521;
+var baseLat = 45.89546589994886;
+var delta = 0.001;
+function addEntity(i, j) {
+    var lon1 = baseLon + delta*i;
+    var lon2 = baseLon + delta*i + delta;
+
+    var lat1 = baseLat + delta*j;
+    var lat2 = baseLat + delta*j + delta;
+    var a = Cesium.Cartesian3.fromDegrees(lon1, lat1);
```

Comment:
Try looking at this with beginners eyes. Can be do better with variable names here?  Is this north/source/east/west?  Why are these polygons instead of rectangles?